x/employeestore/alias.go

```diff
@@ -0,0 +1,29 @@
+package nameservice
+
+import (
+	"github.com/cosmos/sdk-tutorials/nameservice/x/employeestore/internal/keeper"
+	"github.com/cosmos/sdk-tutorials/nameservice/x/employeestore/internal/types"
+)
+
+const (
+	ModuleName = types.ModuleName
+	RouterKey  = types.RouterKey
+	StoreKey   = types.StoreKey
+)
+
+var (
+	NewKeeper        = keeper.NewKeeper
+	NewQuerier       = keeper.NewQuerier
+	NewMsgSetName    = types.NewEmpStore
+	NewWhois         = types.NewWhois
+	ModuleCdc        = types.ModuleCdc
+	RegisterCodec    = types.RegisterCodec
+)
+
+type (
+	Keeper          = keeper.Keeper
+	MsgSetName      = types.StoreEmp
```

Comment:
Naming should be according to original struct name,please change across the code